py3status/modules/file_status.py

```diff
@@ -137,14 +147,15 @@ def file_status(self):
             )
 
         if self.thresholds:
+            self.py3.threshold_get_color(count_path, 'path')
             self.py3.threshold_get_color(count_path, 'paths')
 
         return {
             'cached_until':
             self.py3.time_in(self.cache_timeout),
             'full_text': self.py3.safe_format(
                 self.format, {
-                    'paths': count_path,
+                    'path': count_path,
```

Comment:
@cyrinux Can you add `paths: count_path` here? This trips because it's a list. Deprecation `replace_placeholder` works on `{paths}` but not `\?if=paths`. Ty. Sorry. Xoxo.

Reply:
Done. Xoxo ^^